app/models/wheelchair_maneuver.rb

```diff
@@ -37,9 +37,9 @@ class WheelchairManeuver < ActiveRecord::Base
 
   def set_score
     score = 200
-    POINT_VALUES.each do |key, value|
-      if self.send(key) == false
-        score -= value
+    POINT_VALUES.each_pair do |attribute, point_value|
+      unless self.send(attribute)
```

Comment:
Or, of course, we could build up from 0 rather than building down from 200. If you still don't like `unless`.